apollo-adminservice/src/main/scripts/startup.sh

```diff
@@ -53,6 +56,35 @@ if [ "$windows" == "1" ] && [[ -n "$JAVA_HOME" ]] && [[ -x "$JAVA_HOME/bin/java"
     echo "Windows new JAVA_HOME is: $JAVA_HOME"
 fi
 
+if [[ -n "$JAVA_HOME" ]] && [[ -x "$JAVA_HOME/bin/java" ]];  then
+    _java="$JAVA_HOME/bin/java"
+else
+    echo "no executable java command found"
+    exit 1
+fi
+
+if [[ "$_java" ]]; then
+    version=$("$_java" -version 2>&1 | awk -F '"' '/version/ {print $2}')
+    echo version "$version"
+    version=$(echo "$version" | awk -F. '{printf("%03d%03d",$1,$2);}')
+    # now version is of format 009003 (9.3.x)
+    if [ $version -ge 011000 ]; then
+        echo "java version >= 11"
+        JAVA_OPTS="$JAVA_OPTS -Xlog:gc*:$LOG_DIR/gc.log -Xlog:safepoint -Xlog:gc+heap=trace"
```

Comment:
is there any equivalent set up for `-XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=5 -XX:GCLogFileSize=5M` in jdk 9+?

Reply:
Here is the documentation that I referred to do the migration, it says **Log rotation is handled by the framework.** and I assumed that it's enable by default, am I right?

![image](https://user-images.githubusercontent.com/15965696/55277379-6fc68300-533a-11e9-82c8-922ad26d6ec7.png)

[here](https://docs.oracle.com/javase/9/tools/java.htm#JSWOR-GUID-31503FCE-93D0-4175-9B4F-F6A738B2F4C4) is the documentation